mkdocs.yml

```diff
@@ -76,6 +76,7 @@ nav:
       - design/TrafficSimulator.md
       - design/ZeroMQ.md
       - design/Metrics.md
+      - design/VehicleDynamics.md
```

Comment:
The index of the design/TierIVScenarioFormatVersion2.md is missing.

Reply:
Thank you!
I add in https://github.com/tier4/scenario_simulator_v2/pull/540/commits/a8f6014bc4b86dc9272b9b6d93195be0765696e9